### DISCUSSION TITLE
Catppuccin

### DISCUSSION THREAD

**Discussion by er2de2** [bmaroti9/Overmorrow#8] [Q&A] [answered]
Is it possible for you to add catppuccin mocha color scheme to your application?

**Comment by bmaroti9** [comment, marked as answer]
Hmmm, I've had a look at this catpuccin theme and it looks interesting. However as Overmorrow get's the color palettes from the images themselves, it would look out of place to use the catpuccin theme. I might consider using it for the light and dark themes though, maybe.
